blocks/editable/index.js

```diff
@@ -438,12 +453,10 @@ export default class Editable extends wp.element.Component {
 
 				<TinyMCE
 					tagName={ tagName }
+					onConfig={ this.onConfig }
```

Comment:
Since we only ever use this in `<TinyMCE />` for initializing the instance, could we simply prepare the `settings` ahead of time and avoid the callback?

Thinking something like:

```js
let settings = {
	forced_root_block: this.props.inline ? false : 'p'
};

if ( this.props.onConfig ) {
	settings = this.props.onConfig( settings );
}

return (
	<TinyMCE
		settings={ settings }
		{ /* ... */ }
	/>
);
```